src/pybind/rados/rados.pyx

```diff
@@ -1236,8 +1236,19 @@ Rados object in state %s." % self.state)
 
     def mon_command(self, cmd, inbuf, timeout=0, target=None):
         """
+        Send a command to the mon.
+
         mon_command[_target](cmd, inbuf, outbuf, outbuflen, outs, outslen)
-        returns (int ret, string outbuf, string outs)
+
+        :return: (int ret, string outbuf, string outs)
+
+        Example:
+
+        >>> import json
```

Comment:
`>>> import rados` is missing?

Reply:
well, technically, we're inside of the `rados` module, thus we should not import `rados` here.

Reply:
`>>>` is the prompt of the Python shell. How are we inside the `rados` module when we're in the shell? But OK.